Terasology/ClimateConditions
# feat: Show visual representation of temperature/humidity maps

This pull request adds a screen that displays a portion of the heat/temperature maps, with the values at each pixel represented as colors from black to white so that the maps can be viewed and debugged more easily.

### How to Test
The console command to view the maps is `showMap [temperature/humidity] [height at which to look]`.

When testing, note that the temperature decreases as the height increases, and the maps should reflect that change - when you enter a larger height value, the map should become darker (colder). Also know that the generated maps will just look like noise; temperature and humidity values are not currently based on the world's features.
